.shippable.yml

```diff
@@ -4,10 +4,10 @@ compiler: gcc
 
 env:
     global:
-        - SDK=0.9.1
+        - SDK=0.9.2
         - SANITYCHECK_OPTIONS=" --inline-logs -R"
         - SANITYCHECK_OPTIONS_RETRY="${SANITYCHECK_OPTIONS} --only-failed --outdir=out-2nd-pass"
-        - ZEPHYR_SDK_INSTALL_DIR=/opt/sdk/zephyr-sdk-0.9.1
+        - ZEPHYR_SDK_INSTALL_DIR=/opt/sdk/zephyr-sdk-0.9.2-rc4
```

Comment:
still not ready, will fail on many architectures due to some issues in Qemy, rc5 will come soon.

Reply:
Thanks for the heads up, I'll split up the patchset to get this merged then.